lapce-data/src/config.rs

```diff
@@ -988,23 +987,51 @@ impl notify::EventHandler for ConfigWatcher {
         if let Ok(event) = event {
             match event.kind {
                 notify::EventKind::Create(_)
-                | notify::EventKind::Modify(ModifyKind::Data(DataChange::Content))
+                | notify::EventKind::Modify(_)
                 | notify::EventKind::Remove(_) => {
-                    *self.delay_handler.lock() = Some(());
-                    let delay_handler = self.delay_handler.clone();
-                    let event_sink = self.event_sink.clone();
-                    std::thread::spawn(move || {
-                        std::thread::sleep(std::time::Duration::from_millis(500));
-                        if delay_handler.lock().take().is_some() {
-                            let _ = event_sink.submit_command(
-                                LAPCE_UI_COMMAND,
-                                LapceUICommand::ReloadConfig,
-                                Target::Auto,
-                            );
+                    let short = event
+                        .paths
+                        .iter()
+                        .filter_map(|p| p.file_name())
+                        .any(|p| p == "settings.toml" || p == "keymaps.toml");
+
+                    if short {
```

Comment:
We don't need a specical case for settings.toml and keymaps.toml.

They need to wait for 500ms as well.